etrago/tools/plot.py

```diff
@@ -1607,60 +1673,78 @@ def calc_network_expansion(network, method="abs", ext_min=0.1):
 
     Returns
     -------
-    all_network : :class:`pypsa.Network
+    network_c : :class:`pypsa.Network
```

Comment:
The parameter is named `network`, so you should also call it like this in the docstring

Reply:
Agree